src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/RaCandidateRepository.php

```diff
@@ -53,6 +78,70 @@ public function removeByIdentityId(IdentityId $identityId)
         $this->getEntityManager()->flush();
     }
 
+    /**
+     * @param Institution $institution
+     * @param InstitutionCollection $raInstitutions
+     * @return void
+     */
+    public function removeInstitutionsNotInList(Institution $institution, InstitutionCollection $raInstitutions)
+    {
+        $raCandidates = $this->createQueryBuilder('rac')
+            ->where('rac.raInstitution = :raInstitution')
+            ->andWhere('rac.institution NOT IN (:institutions)')
+            ->setParameter('raInstitution', $institution)
+            ->setParameter('institutions', $raInstitutions)
```

Comment:
This results in error:
```
"Doctrine\\DBAL\\DBALException: An exception occurred while executing 'SELECT r0_.id AS id_0, r0_.identity_id AS identity_id_1, r0_.ra_institution AS ra_institution_2, r0_.institution AS institution_3, r0_.name_id AS name_id_4, r0_.common_name AS common_name_5, r0_.email AS email_6 FROM ra_candidate r0_ WHERE r0_.ra_institution = ? AND r0_.institution NOT IN (?)' with params [\"institution-a.example.com\", {\"institutions\":[\"institution-a.example.com\",\"institution-d.example.com\"]}]:\n\nCatchable Fatal Error: Object of class Surfnet\\Stepup\\Identity\\Collection\\InstitutionCollection could not be converted to string"
```

Reply:
Converting to array seems to fix this for me

Reply:
Fixed